plugins/qeta-backend/migrations/20230314-search-index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function up(knex) {
+  await knex.schema.raw(
+    `CREATE INDEX questions_search_content_index ON questions USING GIN (to_tsvector('english', title || ' ' || content))`,
```

Comment:
I think this does not work in sqlite? Maybe put this also behind database type check for postgres, see the other comment.

Reply:
Yeah it only works for Postgres. I'll try doing the conditional approach 👍